oauth2_http/javatests/com/google/auth/oauth2/ServiceAccountJwtAccessCredentialsTest.java

```diff
@@ -172,29 +171,6 @@ public void getAuthenticationType_returnsJwtAccess() throws IOException {
     assertEquals(credentials.getAuthenticationType(), "JWTAccess");
   }
 
-  @Test
```

Comment:
restore these tests for JWTCredential? Since you have moved the logic there

Reply:
These tests are moved to ServiceAccountCredentialsTest.